rootfs/scheduler/k8s.py

```diff
@@ -25,79 +29,85 @@
         "image": "$image"
       }
     ],
-    "restartPolicy":"Never"
+    "restartPolicy": "Never"
   }
-}'''
-
-RC_TEMPLATE = '''{
-   "kind":"ReplicationController",
-   "apiVersion":"$version",
-   "metadata":{
-      "name":"$name",
-      "labels":{
-         "name":"$id"
-      }
-   },
-   "spec":{
-      "replicas":$num,
-      "selector":{
-         "name":"$id",
-         "version":"$appversion",
-         "type":"$type"
-      },
-      "template":{
-         "metadata":{
-            "labels":{
-               "name":"$id",
-               "version":"$appversion",
-               "type":"$type"
-            }
-         },
-         "spec":{
-            "containers":[
-               {
-                  "name":"$containername",
-                  "image":"$image"
-               }
-            ]
-         }
-      }
-   }
-}'''
-
-SERVICE_TEMPLATE = '''{
-   "kind":"Service",
-   "apiVersion":"$version",
-   "metadata":{
-      "name":"$name",
-      "labels":{
-         "name":"$label"
-      }
-   },
-   "spec":{
-      "ports": [
-        {
-          "port":80,
-          "targetPort":$port,
-          "protocol":"TCP"
+}
+"""
+
+RC_TEMPLATE = """\
+{
+  "kind": "ReplicationController",
+  "apiVersion": "$version",
+  "metadata": {
+    "name": "$name",
+    "labels": {
```

Comment:
IIRC, we need to start using the `app` label here, too. It should be `"app": "$id"`. I think, though, that this should be filed as a separate ticket so that we can discuss what labels ought to be put on here.